package.json

```diff
@@ -9,6 +9,8 @@
   "main": "server.js",
   "scripts": {
     "start:server": "nodemon server.js",
+    "lint:precommit": "lint-staged",
```

Comment:
This will run only on staging file can we have a better name like lint:stage

Reply:
rename to `lint:stage`